platforms/common/spi.cpp

```diff
@@ -138,7 +142,8 @@ bool SPIBusIterator::next()
 
 			case FilterType::ExternalBus:
 				if (bus_data.is_external) {
-					uint16_t cs_index = _devid_driver_index - 1;
+					// Note: chip-select index is starting from 1 in CLI, -1 means not defined
```

Comment:
why this?

Reply:
Therei is not dual use of _devid_driver_index anymore.. check out platforms/common/i2c_spi_buses.cpp